src/bentoml/_internal/cloud/client.py

```diff
@@ -8,47 +8,42 @@
 from ..configuration import BENTOML_VERSION
-from .schemas import BentoRepositorySchema
-from .schemas import BentoSchema
-from .schemas import BentoWithRepositoryListSchema
-from .schemas import ClusterFullSchema
-from .schemas import ClusterListSchema
-from .schemas import CompleteMultipartUploadSchema
-from .schemas import CreateBentoRepositorySchema
-from .schemas import CreateBentoSchema
-from .schemas import CreateDeploymentSchema
-from .schemas import CreateModelRepositorySchema
-from .schemas import CreateModelSchema
-from .schemas import DeploymentListSchema
-from .schemas import DeploymentSchema
-from .schemas import FinishUploadBentoSchema
-from .schemas import FinishUploadModelSchema
-from .schemas import ModelRepositorySchema
-from .schemas import ModelSchema
-from .schemas import ModelWithRepositoryListSchema
-from .schemas import OrganizationSchema
-from .schemas import PreSignMultipartUploadUrlSchema
-from .schemas import UpdateBentoSchema
-from .schemas import UpdateDeploymentSchema
-from .schemas import UserSchema
-from .schemas import schema_from_json
-from .schemas import schema_from_object
-from .schemas import schema_to_json
+from .schemas.schemasv1 import BentoRepositorySchema
+from .schemas.schemasv1 import BentoSchema
+from .schemas.schemasv1 import BentoWithRepositoryListSchema
+from .schemas.schemasv1 import ClusterFullSchema
+from .schemas.schemasv1 import ClusterListSchema
+from .schemas.schemasv1 import CompleteMultipartUploadSchema
+from .schemas.schemasv1 import CreateBentoRepositorySchema
+from .schemas.schemasv1 import CreateBentoSchema
+from .schemas.schemasv1 import CreateDeploymentSchema as CreateDeploymentSchemaV1
+from .schemas.schemasv1 import CreateModelRepositorySchema
+from .schemas.schemasv1 import CreateModelSchema
+from .schemas.schemasv1 import DeploymentFullSchema
+from .schemas.schemasv1 import DeploymentListSchema
+from .schemas.schemasv1 import FinishUploadBentoSchema
+from .schemas.schemasv1 import FinishUploadModelSchema
+from .schemas.schemasv1 import ModelRepositorySchema
+from .schemas.schemasv1 import ModelSchema
+from .schemas.schemasv1 import ModelWithRepositoryListSchema
+from .schemas.schemasv1 import OrganizationSchema
+from .schemas.schemasv1 import PreSignMultipartUploadUrlSchema
+from .schemas.schemasv1 import UpdateBentoSchema
+from .schemas.schemasv1 import UpdateDeploymentSchema
+from .schemas.schemasv1 import UserSchema
+from .schemas.schemasv2 import CreateDeploymentSchema as CreateDeploymentSchemaV2
+from .schemas.schemasv2 import UpdateDeploymentSchema as UpdateDeploymentSchemaV2
+from .schemas.utils import schema_from_json
+from .schemas.utils import schema_from_object
+from .schemas.utils import schema_to_json
 
 logger = logging.getLogger(__name__)
 
 
-class RestApiClient:
-    def __init__(self, endpoint: str, api_token: str) -> None:
+class BaseRestApiClient:
+    def __init__(self, endpoint: str, session: requests.Session) -> None:
```

Comment:
Nit: can we update this with httpx instead? We r removing requests soon

Reply:
Um, should we merge these after this PR? 
https://github.com/bentoml/BentoML/pull/4224
https://github.com/bentoml/BentoML/pull/4227

Reply:
I don't see why not to do it now. Probably better to do it now.

Reply:
Rebased https://github.com/bentoml/BentoML/pull/4224 here, httpx should be good to go!